docker-storage-setup.sh

```diff
@@ -628,8 +628,16 @@ check_wipe_block_dev_sig() {
 canonicalize_block_devs() {
   local devs=$1 dev
   local devs_abs dev_abs
+  local dest_dev
 
   for dev in ${devs}; do
+    # If the device name is a symlink, follow it and use the target
+    if [ -h "$dev" ];then
+      if ! dest_dev=$(readlink -e $dev);then
+        Fatal "$dest_dev failed to resolve linked device path $dev"
```

Comment:
When error happens $dest_dev will be empty string. Right? So it will not help using dest_dev in error message. So how about just keeping it simple as.

`Fatal "Failed to resolve symbolic link $dev"`